package.json

```diff
@@ -105,9 +104,9 @@
     "@types/cors": "^2.8.17",
     "@types/express": "^4.17.17",
     "@types/express-rate-limit": "^5.1.3",
-    "@types/graphql-depth-limit": "^1.1.6",
-    "@types/i18n": "^0.13.10",
-    "@types/jsonwebtoken": "^9.0.5",
+    "@types/graphql-depth-limit": "^1.1.3",
+    "@types/i18n": "^0.13.3",
```

Comment:
why are these packages being downgraded? sync your local repo with develop branch of talawa-api main repo and make a new commit